docs/set_up_admin_tails.rst

```diff
@@ -136,18 +136,37 @@ signed with the release signing key:
 .. code:: sh
 
     cd ~/Persistent/securedrop/
-    git checkout 1.6.0
+    git fetch --tags
+    gpg --keyserver hkps://keys.openpgp.org --recv-key \
+    "2224 5C81 E3BA EB41 38B3 6061 310F 5612 00F4 AD77"
     git tag -v 1.6.0
 
 You should see ``Good signature from "SecureDrop Release Signing Key"`` in the
-output of that last command along with the fingerprint above.
+output.
+
+The output should include the following two lines:
+
+.. code:: sh
+
+    gpg:                using RSA key 22245C81E3BAEB4138B36061310F561200F4AD77
+    gpg: Good signature from "SecureDrop Release Signing Key"
+
 
 .. important::
 
    If you do not see the message above, signature verification has failed
    and you should **not** proceed with the installation. If this happens,
    please contact us at securedrop@freedom.press.
 
+Verify that each character of the fingerprint matches what is on the 
+screen of your workstation. If it does, you can check out the new release:
+
+.. code:: sh
+
+    git checkout 1.6.0
```

Comment:
After this step, we want to insert the language we've been using in the upgrade guides, i.e.

```
.. important:: If you do see the warning "refname '1.6.0' is ambiguous" in the
  output, we recommend that you contact us immediately at securedrop@freedom.press
  (`GPG encrypted <https://securedrop.org/sites/default/files/fpf-email.asc>`__).
```

We could move this into an include to reduce the amount of copy/paste.

Reply:
Just out of curiosity, what does this warning suggest? Is it a security issue? Why should a user contact immediately?

Reply:
We assume that `1.6.0` would refer to a signed git tag on the repository at https://github.com/freedomofpress/securedrop . When running `git checkout 1.6.0`, however, git will also happily checkout a branch, rather than a tag, if the branch is named "1.6.0". In that case, it's not clear to git which the user wanted. We always want the checkout to result in a tag, but there's no flag we can pass to git to make a branch name unacceptable as a refspec for the checkout command.

Reply:
(Just a note that the include `refname-warning.txt` is still missing from this PR, in case you forgot to push a commit.)